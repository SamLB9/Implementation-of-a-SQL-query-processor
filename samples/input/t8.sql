SELECT Flight.FlightID, SUM(Flight.Price * Flight.Duration) FROM Flight GROUP BY Flight.FlightID;