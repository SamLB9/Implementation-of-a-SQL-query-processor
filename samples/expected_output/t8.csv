1, 147000
2, 13500
3, 192000
4, 360000
5, 270000